README.rst

```diff
@@ -3,10 +3,10 @@ SQLAlchemy-Utc
 
 .. image:: https://badge.fury.io/py/SQLAlchemy-Utc.svg?
    :target: https://pypi.python.org/pypi/SQLAlchemy-Utc
-.. image:: https://travis-ci.org/spoqa/sqlalchemy-utc.svg?branch=master
-   :target: https://travis-ci.org/spoqa/sqlalchemy-utc
-.. image:: https://codecov.io/github/spoqa/sqlalchemy-utc/coverage.svg?branch=master
-   :target: https://codecov.io/github/spoqa/sqlalchemy-utc?branch=master
+.. image:: https://travis-ci.com/ItachiSan/sqlalchemy-utc.svg?branch=master
+   :target: https://travis-ci.com/ItachiSan/sqlalchemy-utc
+.. image:: https://codecov.io/github/ItachiSan/sqlalchemy-utc/coverage.svg?branch=master
+   :target: https://codecov.io/github/ItachiSan/sqlalchemy-utc?branch=master
```

Comment:
Could you remove this change?

Reply:
I can revert my username for sure :)
I used that for verifying the Travis-CI and Codecov integration!
travis-ci.org however became .com, so I'll keep that :)